README.md

```diff
@@ -14,10 +14,10 @@ const Greeting = ({ name = 'World' }) => (
   <p>Hello, {name}!</p>
 );
 
-register(Greeting, 'x-greeting', ['name'], { shadow: true, mode: 'open', adoptedStyleSheets: [] });
-//          ^            ^           ^             ^               ^            ^
-//          |      HTML tag name     |       use shadow-dom        |    use adoptedStyleSheets
-//   Component definition      Observed attributes     Encapsulation mode for the shadow DOM tree
+register(Greeting, 'x-greeting', ['name'], { shadow: true, mode: 'open', adoptedStyleSheets: [], serializable: true });
+//          ^            ^           ^             ^               ^            ^                    ^
+//          |      HTML tag name     |       use shadow-dom        |    use adoptedStyleSheets     |
+//   Component definition      Observed attributes     Encapsulation mode for the shadow DOM tree   Enable declarative shadow DOM
```

Comment:
This isn't aligned and `serializable` does not enable DSD. It simply indicates that the shadow root is serializable _which may_ be used for DSD (but can serve other purposes too).